### DISCUSSION TITLE
Any plans to update?

### DISCUSSION THREAD

**Discussion by sH1222J** [kotatogram/kotatogram-desktop#205] [EN 2. Q&A]
Telegram desktop version is 2.9.0 and Kotatogram desktop version is 2.7.4.
I am inquiring if there is any plan to update it.

**Comment by ilya-fedin** [comment]
Yeah, as soon as hardware rendering will be stabilized in tdesktop

**Reply by 3k2** [reply]
> Yeah, as soon as hardware rendering will be stabilized in tdesktop

![image](https://user-images.githubusercontent.com/11908261/144600121-f10586e7-ff3b-4f14-90fa-576cf778a1ea.png)

I hope you update soon coz soon kotatogram doesn't work anymore.

**Reply by EricKotato** [reply]
@3k2 you can update to beta version 1.4.4, it's on updated TD sources, so it shouldn't have this error.

**Reply by zelurker** [reply]
Yeah that's something I just noticed here, the official desktop client has a slow start with an opengl error in linux :
QPainter::begin: Paint device returned engine == 0, type: 3
and your client, kotatogram doesn't have it ! So congratulations, even if disabling opengl is another way to fix the problem on the official client, not sure opengl is really useful here... !

**Reply by ilya-fedin** [reply]
QPainter is software rendering, not opengl. The error is harmless, actually.

**Comment by zelurker** [comment]
Yeah I just noticed the message is still here even when opengl is disabled.
Well there is an opengl initialization bug at least, the original client
takes a few seconds to initialize with some "frozen" are where its window
should be, and after this noticeable delay it starts normally without any
noticeable impact.
No message and no delay with kotatogram, so it's an opportunity for me to
test it !

Le sam. 10 févr. 2024 à 14:03, ilya-fedin ***@***.***> a
écrit :
